src/models/saved_export/render.ts

```diff
@@ -185,6 +188,11 @@ async function renderAsCSV(events) {
   return await processing;
 }
 
+function unixToIso(unixTimestamp: number) {
+  const date = new Date(unixTimestamp);
+  return date.toISOString().slice(0, 19).replace("T", " ");
```

Comment:
what happens here if it's a bad date?

Reply:
will add a validation here to check if the timestamp provided is valid or not by checking upper and lower bounds and throw an error if the timestamp is invalid